src/generate-id.js

```diff
@@ -0,0 +1,24 @@
+const crypto = require('crypto')
+
+/**
+ * Generate a unique ID with base64 url encoding
+ * @param {number} [numBytes=18] - Number of random bytes to include in the ID
+ * @returns {string} Random string with 4/3*numBytes characters
+ */
+function generateId (numBytes = 18) {
```

Comment:
one day we should really use an npm registry or something for these things. Or a monorepo.